geomesa-blobstore/geomesa-blobstore-api/src/main/java/org/locationtech/geomesa/blob/api/Blob.java

```diff
@@ -0,0 +1,49 @@
+/***********************************************************************
+* Copyright (c) 2013-2016 Commonwealth Computer Research, Inc.
+* All rights reserved. This program and the accompanying materials
+* are made available under the terms of the Apache License, Version 2.0
+* which accompanies this distribution and is available at
+* http://www.opensource.org/licenses/apache2.0.php.
+*************************************************************************/
+
+package org.locationtech.geomesa.blob.api;
+
+
+import com.google.common.base.Objects;
+
+public class Blob {
+
+    private String id;
+    private String localName;
+    private byte[] payload;
+
+    public Blob() {
```

Comment:
is there a reason to have the public no arg constructor? There aren't any setters. If not mark the member variables final.